code/drasil-gool/lib/Drasil/GOOL/RendererClassesCommon.hs

```diff
@@ -148,11 +148,15 @@ class RenderValue r where
   -- calls.
   call :: Maybe Library -> Maybe Doc -> MixedCall r
 
-  valFromData :: Maybe Int -> Maybe Integer -> VSType r -> Doc -> SValue r
+  valFromData :: Maybe Int -> LitValue -> VSType r -> Doc -> SValue r
```

Comment:
The previous type was more precise! Perhaps `LitValue` is not the best way to go, but instead a synonym for `Maybe a` would be, where the type could be more precise.